ee/modules/030-cloud-provider-vcd/templates/cloud-data-discoverer/deployment.yaml

```diff
@@ -104,11 +104,19 @@ spec:
             secretKeyRef:
               key: username
               name: vcd-credentials
+              optional: true
         - name: VCD_PASSWORD
           valueFrom:
             secretKeyRef:
               key: password
               name: vcd-credentials
+              optional: true
+        - name: VCD_TOKEN
```

Comment:
i think we need `if` template because refresh token can be skipped. Please check it